src/main/java/de/pellepelster/jenkins/walldisplay/WallDisplayViewAction.java

```diff
@@ -8,44 +8,43 @@
 
 /**
  * Action providing the link to the actual walldisplay page
- *
+ * 
  * @author Christian Pelster
  */
 public class WallDisplayViewAction implements Action {
 
-    private static final String ENCODING = "UTF-8";

```

Comment:
Please do not reformat existing code. In general a PR should include the minimum possible number of diff lines.